src/inference/dev_api/openvino/runtime/internal_properties.hpp

```diff
@@ -67,6 +67,26 @@ static constexpr Property<ov::threading::IStreamsExecutor::ThreadBindingType, Pr
  */
 static constexpr Property<size_t, PropertyMutability::RW> threads_per_stream{"THREADS_PER_STREAM"};
 
+/**
+ * @brief It contains compiled_model_runtime_properties information to make plugin runtime can check whether it is
+ * compatible with the cached compiled model, the result is returned by get_property() calling.
+ *
+ * The information details are defined by plugin itself, each plugin may require different runtime contents.
+ * For example, CPU plugin will contain OV version, while GPU plugin will contain OV and GPU driver version, etc.
+ * Core doesn't understand its content and only read it from plugin and write it into blob header.
+ *
+ * @ingroup ov_dev_api_plugin_api
+ */
+static constexpr Property<std::string, PropertyMutability::RO> compiled_model_runtime_properties{
```

Comment:
May I know any benefits if we choose `std::vector<std::string>` rather than `std::string`?
From my understanding, core level doesn't need understand its contents and only read it from blob file's header or send it to plugin side, so `std::string` is enough, right?

Reply:
In plugin side, we adopt `ov::AnyMap` and convert it to or from `std::string`, seems it is not too complex?